.circleci/config.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           command: yarn build
       - run:
           name: Yarn Deploy Staging
-          command: aws s3 sync build/ s3://staging-semgrep-dev/docs
+          command: aws s3 sync build/ s3://staging-semgrep-dev/docs --delete
```

Comment:
I have not run this myself, but my instinct says we wouldn't need to do this for staging too...

Reply:
We barely use that staging bucket so I think it doesn’t matter much. I added it here mostly for consistency between staging and the other command.